services/121-service/src/seed-data/program/program-nlrc-ocw.json

```diff
@@ -90,7 +90,7 @@
   "aboutProgram": {
     "en": "This is the OCW program."
   },
-  "fullnameNamingConvention": ["firstName", "lastName"],
+  "fullnameNamingConvention": ["fullName"],
```

Comment:
There should now also be a program question called fullName. During testing I found that a payment only works for PV for this specific reason.

Reply:
It is there... Let's check this in a meeting, maybe I am missing something.

Reply:
Tested with Mock Service on both OCW and PV programs, doing 2 payments in each. All works. I did find that the test data CVSs still contained first and last name. So I corrected that. Will probably conflict with Piotr's work by now :)